home/modules/ROOT/pages/mobile.adoc

```diff
@@ -1,69 +1,100 @@
-= Couchbase Mobile - Embedded Database on the Edge
-:page-layout: landing-page-top-level-sdk
-:page-role: tiles
+= Couchbase Mobile 
+// :page-layout: landing-page-top-level-sdk
+// :page-role: tiles 
+:page-toclevels: 2
 :page-rank: 75
-:!sectids:
 
-include::partial$info-banner.adoc[]
```

Comment:
Nitpick - this include, though it currently doesn't appear to do anything, always calls the "Info Banner" partial, which can then either:

* display nothing (as currently)
* display an info-banner with any site-wide info that we want to advertise.

It's not urgent for now (we'd do a sweep before adding a new banner) but just to explain what it was for 😄